administrator/language/pl-PL/com_scheduler.sys.ini

```
@@ -10,7 +10,7 @@ COM_SCHEDULER_MANAGER_TASK="Menedżer zadania"
 COM_SCHEDULER_MANAGER_TASK_EDIT="Edytuj zadanie"
```

Skoro już poprawiamy, to przy okazji także i to
```suggestion
COM_SCHEDULER_MANAGER_TASK_EDIT="Dostosuj zadanie"
```